projects/admin/src/app/service/items.service.ts

```diff
@@ -57,11 +57,11 @@ export class ItemsService {
   }
 
   doValidateRequest(item, transactionLibraryPid) {
-    const url = '/api/item/validate';
+    const url = '/api/item/validate_request';
```

Comment:
URL can be put directly in the post method, unnecessary to store it in a constant.